app/actuators/ResultCard.js

```diff
@@ -627,7 +627,7 @@ export default class ResultCard extends Component {
 		}
 
 		return (
-			<div className="rbc rbc-resultcard" ref={(node) => { this.listContainer = node; }}>
+			<div className=`rbc rbc-resultcard ${this.props.className ? this.props.className : ""}` ref={(node) => { this.listContainer = node; }}>
```

Comment:
You need to wrap className values with`{ ... }`.

Reply:
Updated.
Thank you for fixing it in reactivebase. :sweat_smile: